src/runtime/virtcontainers/README.md

```diff
@@ -320,6 +323,81 @@ Clear Containers kernel. If the driver is missing,  you can add it to your
 custom container kernel using the [osbuilder](https://github.com/clearcontainers/osbuilder)
 tooling.
 
+## How to pass a device using VFIO-AP passthrough
+
+IBM Z mainframes (s390x) use the AP (Adjunct Processor) bus for their Crypto
+Express hardware security modules. Such devices can be passed over VFIO, which
+is also supported in Kata. Pass-through happens separated by adapter and
+domain, i.e. a passable VFIO device has one or multiple adapter-domain
+combinations.
+
+1. You must follow the [kernel documentation for preparing VFIO-AP passthrough](https://www.kernel.org/doc/html/latest/s390/vfio-ap.html).
+In short, your host kernel should have the following enabled or available as
+module (in case of modules, load the modules accordingly, e.g. through
+`modprobe`). If one is missing, you will have to update your kernel
```

Comment:
uhmm were you able to load a module in a container using `modprobe` ?

Reply:
> In short, your host kernel should have the following enabled

No, this is host kernel

Reply:
ahh sorry , I missed the word `host`, thanks